server/helpers/createUser.js

```diff
@@ -0,0 +1,28 @@
+import hash from 'bcrypt-nodejs';

```

Comment:
Can you kindly explain why you need this file and why you are using static `user` data?

Reply:
The reason is so that I can use that user's token to create, modify, delete and get all entries.
I need that user's token to test diary entry endpoints.